### ISSUE TITLE
Callback not called if label change does not result in date-range change

### ISSUE THREAD

**Issue by cloventt** [dangrossman/daterangepicker#2121]
tl;dr If two labels result in the same date-range, the display does not work and the callback is not fired.

Setup:
- One custom range called _Last 30 Days_, calculated as 00:00 30 days ago to 00:00 today.
- Another custom range called _Last Month_, calculated as 00:00 of last month to 00:00 of the current month).
- System time is any time on July 1st.

Effect:
- Clicking drop-down shows both custom ranges, with _Last 30 Days_ first.
- _Last 30 Days_ is highlighted by default.
- The callback is not called when selecting _Last Month_, even though a different label has been selected.
- _Last Month_ cannot be selected.

I expected the callback to be fired if the label changes, even if the selected dates do not change. The issue occurs because on July 1st, both of these time ranges result in the exact same start and end dates, so the date actually doesn't change under the hood.

I've created a fiddle demonstrating the issue (you will need to change your system time to July 1st or some other date that creates the problem). https://jsfiddle.net/qpv92c3r/3/

This breaks my use-case, as my system has the option of saving the labels as well as the time ranges. This means that a user is unable to save a report with the label _Last Month_ on the first day of July.

**Comment by cloventt** [comment]
For future searchers, a workaround is to instead bind to the `apply.daterangepicker` [event](https://www.daterangepicker.com/#events) which is called when a value is selected, even if the underlying value does not change.
